testsuite/integration/basic/src/test/java/org/jboss/as/test/integration/messaging/jms/definitions/JMSResourceManagementTestCase.java

```diff
@@ -75,9 +77,12 @@ public static JavaArchive createArchive() {
                 .addAsManifestResource(
                         MessagingBean.class.getPackage(), "ejb-jar.xml", "ejb-jar.xml")
                 .addAsManifestResource(
-                        EmptyAsset.INSTANCE,
+                        new StringAsset("<beans xmlns=\"https://jakarta.ee/xml/ns/jakartaee\" xmlns:xsi=\"http://www.w3.org/2001/XMLSchema-instance\"\n" +
+"xsi:schemaLocation=\"https://jakarta.ee/xml/ns/jakartaee https://jakarta.ee/xml/ns/jakartaee/beans_3_0.xsd\"\n" +
+"bean-discovery-mode=\"all\">\n" +
+"</beans>"),
                         create("beans.xml"));
-
+        archive.as(ZipExporter.class).exportTo(new File("JMSResourceDefinitionsTestCase.jar"));
```

Comment:
is this a temporary debugging statement?

Reply:
yes

Reply:
correct